src/main/java/uk/gov/justice/digital/hmpps/community/model/ManagedOffender.java

```diff
@@ -0,0 +1,40 @@
+package uk.gov.justice.digital.hmpps.community.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.*;
+
+import java.time.LocalDate;
+
+@ApiModel(description = "ManagedOffender")
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@EqualsAndHashCode
+@ToString
+@Data
+public class ManagedOffender {
+
+    @ApiModelProperty(required = true)
```

Comment:
Could you add a bit more info on the properties

Reply:
I've added full descriptions to this API model, so will merge now. thanks.